checkrebuild

```diff
@@ -22,17 +22,24 @@ get_unofficial_pkgs() {
     comm -23 <(pacman -Qq | sort) <(pacman -Sql "${repos_to_skip[@]}" 2>/dev/null | sort)
 }
 
-filter_executable() {
-    while read -r file; do
-        [[ -f "$file" && -x "$file" ]] && echo "$file"
-    done
+newline2null(){
+    perl -pe 's/\n/\0/'
```

Comment:
you can move it back inline because we use this only once, and only as a cheat because as far as I can see there is no way for pacman to print files separated by `\0` - once converted, we should try to never convert back to newline-separated.